Sources/IndexStoreDB/IndexStoreDB.swift

```diff
@@ -187,6 +187,27 @@ public final class IndexStoreDB {
     }
     return result
   }
+
+  /// Iterates over the name of every symbol in the index.
+  ///
+  /// - Parameter body: A closure to be called for each symbol. The closure should return true to
+  /// continue iterating.
+  @discardableResult
+  public func forEachSymbolName(body: @escaping (String) -> Bool) -> Bool {
+    return indexstoredb_index_symbol_names(impl) { name in
+      body(String(cString: name))
+    }
+  }
+
+  /// Returns a set with every symbol in the index.
+  public func allSymbolNames() -> Set<String> {
```

Comment:
This doesn't need to be a `Set`, `indexstoredb_index_symbol_names` is going to return unique strings.
I'd recommend to change it to array.

Reply:
Done!